e2e/pages/events/uploadC2DocumentsEvent.page.js

```diff
@@ -24,6 +38,20 @@ module.exports = {
     I.fillField(this.fields.description, description);
   },
 
+  async uploadC2SupportingDocument() {
+
+    await I.addAnotherElementToCollection();
+    I.fillField(this.fields.supportingDocuments.name, c2SupportingDocuments.name);
+    I.fillField(this.fields.supportingDocuments.notes, c2SupportingDocuments.notes);
+    I.fillField(this.fields.supportingDocuments.dateAndTime.day, c2SupportingDocuments.dateAndTime.day);
```

Comment:
Note that there is a helper method that offers part of this functionality, `I.fillDate(...)`. Could probably be extended to fill in the time as well.

Reply:
done